.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,38 @@
+### How to submit issue
+Please use this text as a template and replace text in the sections or remove
+the entire section if that does not apply to your issue. For example in case of
+question or feature request, just description with some example is probably
+fine. Also remember to use github's markup form properly, especially in case of
+output or code listing.
+
+### Things you may try first
+(put "x" in "[]" if you already tried following)
+* [] Did you check if this is a duplicate issue?
+* [] Did you test it on the latest FRRouting/frr master branch?
```

Comment:
I believe if you add a space in between the `[]` it shows up as an empty checkbox:

- [ ] xyz